oneanddone/users/models.py

```diff
@@ -124,5 +125,13 @@ def profile_url(self):
         else:
             return reverse('users.profile.details', args=[self.user.id])
 
+    @property
```

Comment:
Please move the property definition up so that it is before `email` to keep the properties in alphabetical order.